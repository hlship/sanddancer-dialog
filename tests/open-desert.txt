exit
e
purloin flashlight
turn it on
e
(now) (#emergency-lights is on)
open wallet
e
s