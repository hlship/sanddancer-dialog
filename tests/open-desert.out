

Middle of Nowhere (in the pickup truck)
You've smashed up against a tall Saguaro, which is all you can see out the
windshield except a few feet of desert sand, then blackness. Hell, you must have
driven off the road and crashed. Ocean keeps telling you this night commute on
this lonely highway's gonna kill you; maybe you should start listening.

On the dash you can see a piece of jade.

The glove compartment is closed.

Through the windshield, you see a little lizard clinging frantically to the
Saguaro.

> exit
The door creaks open with difficulty, rusted joints straining. You get out of
the pickup truck.

Something about the piece of jade makes you want to brood about your grandma's
stories.

> e
Outside the wavering glow of your headlights, it's black. Pitch. Storm must have
rolled in; there's no stars above, no anything but blackness around. Only the
smell of the desert tells you it's still out there.

> purloin flashlight

> turn it on
Your surroundings spring into brightness around you.

Middle of Nowhere
The tire tracks from the south stop abruptly here, but where the hell are you?
The desert sand and clumps of pale sagebrush are all your dimming headlights
pick out before barely reaching the concrete building to the north.

Your poor old pickup truck ticks and groans, smashed gracelessly against a tall
Saguaro; wiffs of evaporating gasoline linger in the chill air.

A brown-colored lizard clings to the cactus.

> e
You heft your flashlight nervously, licking your lips, but decide not to head
out into the desert just yet. Your truck lights are already starting to dim, and
with how dark it is, you're not sure you could find your way back.

> (now) (#emergency-lights is on)
> open wallet
You open your wallet, revealing your driver's license, a receipt from Big
Jimmy's, and a folded photo.

Something about the receipt from Big Jimmy's makes you want to brood about your
shit job.

> e
You walk east for a long time, thoughts turning over like the sand under your
sneakers. You think about your shit job. You think about Ocean and how pissed
she's gonna be that you're not home. Like you tell her not to wait up for you
but she always does anyway.

Some wind kicks up and chucks sand in your face. You blink it out and kinda
realize while you're doing it that you haven't really been paying attention to
where you've been going. Another gust whines through the sagebrush and you
realize with a kind of heavy feeling that with all the dust this wind is kicking
up you can't see a damn thing. You can't see your pickup, you can't see the
electrical tower. Not even your damn footprints. You're lost. Well, shit.

Open Desert
Clouds of dust swirl through the air, and you wish you could see more than
twenty feet in front of you.

Nearby: some dry bones, a stand of cactus, and a dead cow. Nothing to write home
about.

> s
You walk south.

Open Desert
The dust storm rages around you, and every inch of sand looks just like every
other.

You can sort of see some dry bones, a dry streambed, and a stand of cactus.

> 
